CsvParser/csvParser_test.go

```diff
@@ -104,29 +164,141 @@ func Test_CsvParserFiftyFirstEntry(t *testing.T) {
 	}
 }
 
-func Test_MakeJsonObjFile(t *testing.T) {
+func Test_FileHierachy(t *testing.T) {
+	ex, err := os.Executable()
+	if err != nil {
+		panic(err)
+	}
+	originPath := filepath.Dir(ex)
```

Comment:
What exactly is this originPath in these tests? Do the incident JSONs actually get written and if so, where?